Calculator/Calculator/Model/InputManager.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  InputManager.swift
+//  Calculator
+//
+//  Created by Harry, Goat on 2023/02/08.
+//
+
+import Foundation
+
+final class InputManager {
+    var expression = ""
+    var currentNumber = "0"
+    var currentOperator = ""
+    var isCalculated: Bool = false
+    
+    func handleNumbers(input: String) -> String {
+        if currentNumber == "0" || isCalculated {
+            currentNumber = input
+            isCalculated = false
+        } else {
+            currentNumber += input
+        }
+        
+        return currentNumber
+    }
+    
+    func handleDotSign(input: String) -> String {
+        if currentNumber.contains(".") {
+            return currentNumber
+        } else {
+            currentNumber += input
+        }
+        
+        return currentNumber
+    }
+    
+    func handleHundredNumber(input: String) -> String {
+        if currentNumber == "0" {
+            return currentNumber
+        } else {
+            currentNumber += input
+        }
+        
+        return currentNumber
+    }
```

Comment:
[P3] 이 부분도 아래와 같이 변경할 수 있겠네요 :)
```suggestion
    func handleHundredNumber(input: String) -> String {
        if currentNumber != "0" {
            currentNumber += input
        }
        
        return currentNumber
    }
```

Reply:
넵 좋습니다! 수정했습니다

Reply:
이 부분도 마찬가지로 수정하였습니다 !!